diff --git a/tiller/values.yaml b/tiller/values.yaml
index 3865f2ae..f3ec7785 100644
--- a/tiller/values.yaml
+++ b/tiller/values.yaml
@@ -26,7 +26,7 @@ release_group: null
 
 images:
   tags:
-    tiller: gcr.io/kubernetes-helm/tiller:v2.14.1
+    tiller: quay.io/helmpack/tiller:v2.14.1
     dep_check: quay.io/airshipit/kubernetes-entrypoint:v1.0.0
     image_repo_sync: docker.io/docker:17.07.0
   pull_policy: IfNotPresent
diff --git a/tools/deployment/common/005-deploy-k8s.sh b/tools/deployment/common/005-deploy-k8s.sh
index b4fe61b9..0661bb7e 100755
--- a/tools/deployment/common/005-deploy-k8s.sh
+++ b/tools/deployment/common/005-deploy-k8s.sh
@@ -77,7 +77,8 @@ sudo -E apt-get install -y \
     rbd-nbd \
     nfs-common \
     bridge-utils \
-    iptables
+    iptables \
+    conntrack
 
 sudo -E tee /etc/modprobe.d/rbd.conf << EOF
 install rbd /bin/true
@@ -103,7 +104,7 @@ sudo -E chmod +x /usr/local/bin/kubectl
 # Install Helm
 TMP_DIR=$(mktemp -d)
 sudo -E bash -c \
-  "curl -sSL ${URL}/kubernetes-helm/helm-${HELM_VERSION}-linux-amd64.tar.gz | \
+  "curl -sSL https://get.helm.sh/helm-${HELM_VERSION}-linux-amd64.tar.gz | \
     tar -zxv --strip-components=1 -C ${TMP_DIR}"
 
 sudo -E mv "${TMP_DIR}"/helm /usr/local/bin/helm
@@ -121,6 +122,7 @@ sudo -E minikube start \
   --docker-env HTTP_PROXY="${HTTP_PROXY}" \
   --docker-env HTTPS_PROXY="${HTTPS_PROXY}" \
   --docker-env NO_PROXY="${NO_PROXY},10.96.0.0/12" \
+  --extra-config=kubelet.max-pods=300 \
   --network-plugin=cni \
   --extra-config=controller-manager.allocate-node-cidrs=true \
   --extra-config=controller-manager.cluster-cidr=192.168.0.0/16
@@ -179,6 +181,8 @@ EOF
 # which has been removed in Kubernetes 1.16.0.
 # See: https://github.com/helm/helm/issues/6374
 helm init --service-account helm-tiller --output yaml \
+  --stable-repo-url=https://charts.helm.sh/stable \
+  --tiller-image=quay.io/helmpack/tiller:${HELM_VERSION} \
   | sed 's@apiVersion: extensions/v1beta1@apiVersion: apps/v1@' \
   | sed 's@  replicas: 1@  replicas: 1\n  selector: {"matchLabels": {"app": "helm", "name": "tiller"}}@' \
   | kubectl apply -f -
@@ -203,7 +207,8 @@ kubectl --namespace=kube-system wait \
   pod -l app=helm,name=tiller
 EOF
 
-helm init --client-only
+helm init --client-only --stable-repo-url=https://charts.helm.sh/stable \
+  --tiller-image=quay.io/helmpack/tiller:${HELM_VERSION}
 
 # Set up local helm server
 sudo -E tee /etc/systemd/system/helm-serve.service << EOF
